src/main/java/org/synyx/urlaubsverwaltung/department/DepartmentService.java

```diff
@@ -224,6 +224,15 @@ public interface DepartmentService {
      */
     boolean isSignedInUserAllowedToAccessPersonData(Person signedInUser, Person person);
 
+    /**
+     * Check if the given signed in user is allowed to access the data of the given person.
+     *
+     * @param signedInUser to check the permissions
+     * @param department   which data should be accessed
+     * @return {@code true} if the given user may access the data of the given person, else {@code false}
+     */
+    boolean isSignedInUserAllowedToAccessDepartmentData(Person signedInUser, Department department);
```

Comment:
```suggestion
    /**
     * Check if the given person is allowed to access the data of the given department.
     *
     * @param person to check the permissions
     * @param department which data should be accessed
     * @return {@code true} if the given person may access the data of the given person, else {@code false}
     */
    boolean isPersonAllowedToAccessDepartmentData(Person person, Department department);
```

Das hat nichts mit `signedInUser` zu tun. Wir wissen hier nur, dass es sich um eine `Person` handelt.

Die Suggestion **nicht** übernehmen. Da fehlt natürlich alles andere noch im Java Code :-)